topics/epigenetics/tutorials/cut_and_run/tutorial.md

```diff
@@ -0,0 +1,595 @@
+---
+layout: tutorial_hands_on
+
+title: CUT & RUN data analysis
+zenodo_link: https://zenodo.org/record/3862793
+questions:
+- Which binding motif has the transcription factor GATA1?
+- What kind of quality control do I have to do?
+- How to analyse and visualise CUT & RUN data?
+objectives:
+- Apply appropriate analysis and quality control steps for CUT & RUN
+- Apply an enrichment analysis and a robust peak detection
+- Find protein binding motifs
+time_estimation: 3h
+key_points:
+- CUT & RUN can be used to identify binding regions for
+- Several filters are applied to the reads, such as removing those mapped to mitochondria
+- Fragment distribution can help determine whether an CUT & RUN experiment has worked well
+- A robust peak detection removes noise from the data
+contributors:
+- heylf
```

Comment:
ahh is this branch from a recent version of the gtn? I think that was merged a few weeks ago already but if yours is started earlier.. you could rebase it if you want to use it, otherwise don't worry about it!

Reply:
we should update planemo to make this the default @hexylena